colorcoder/EvenCountColorCode.java

```diff
@@ -0,0 +1,23 @@
+package colorcoder;
+
+
+import colorcoder.colorcode.manual.ColorCodeManual;
+import colorcoder.colors.MajorColor;
+import colorcoder.colors.MinorColor;
+
+import static colorcoder.testcolor.TestColorcoder.testNumberToPair;
+import static colorcoder.testcolor.TestColorcoder.testPairToNumber;
+
+
+public class EvenCountColorCode {
+
+    public static void main(String[] args) {
+
+        ColorCodeManual.generateColorCodeManual();
```

Comment:
How is the output of method generateColorCodeManual() verified. Right now it prints onto console and it needs manual verification. This needs to be verified programmatically.